dart/lib/src/utils.dart

```diff
@@ -0,0 +1,5 @@
+bool indexOfMatch(String string, Pattern pattern) =>
```

Comment:
Instead of creating a new file, could you just make this a private function? Having a separate file for 2 lines seems a bit like overkill.

Also, can you rename to something like "matchesPattern"? When I read "indexOfMatch", it sounds like it returns an int.

Reply:
Did it.
Thanks!